### PR TITLE
healthcare prescription proof of concept

### PR DESCRIPTION
**Pull request by garymillergit**

### CODE REVIEW

**Review by xhliu — APPROVED** [review]
👍 This looks great. Glad to see a healthcare contract.

Can u please add a high-level description of what this contract does, besides the existing inline comments?

File: contracts/dummy_prescription.scrypt [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import "util.scrypt";
+
```
Comment (xhliu) [inline contracts/dummy_prescription.scrypt:2]: High-level description can go here.

File: deployments/dummyPrescription.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+// prescriber information
+const prescriberPriv = generatePrivKey();
+const prescriber_nRabin = privKeyToPubKey(prescriberPriv.p, prescriberPriv.q);
+const prescriberSig = sign(
+  prescriptionIDHex,
+  prescriberPriv.p,
+  prescriberPriv.q,
+  prescriber_nRabin
+);
+const privateKeyPrescriberOffice = new bsv.PrivateKey.fromWIF(
```
Comment (garymillergit) [inline deployments/dummyPrescription.js:34]: I think I should call this like prescriptionAuthority

File: tests/js/dummyPrescription.scrypttest.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      tx,
+      privateKeyPharmacy1,
+      dummyPrescription.lockingScript.toASM(),
+      inputSatoshis
+    );
+    // generate tx preImage
+    preimage = getPreimage(
+      tx,
+      dummyPrescription.lockingScript.toASM(),
+      inputSatoshis
```
Comment (garymillergit) [inline tests/js/dummyPrescription.scrypttest.js:175]: If I comment this out, my test still passes, which seems like an issue.

Comment (garymillergit) [reply tests/js/dummyPrescription.scrypttest.js:175]: nvm, the test is expected to fail so the test will pass if anything is wrong.

**Comment by garymillergit** [comment]
@xhliu ready to merge

**Pull request merged**